WordPress/Classes/ViewRelated/Site Creation/Design Selection/Preview/SiteDesignPreviewViewController.swift

```diff
@@ -19,6 +21,17 @@ class SiteDesignPreviewViewController: TemplatePreviewViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         navigationItem.leftBarButtonItem = CollapsableHeaderViewController.closeButton(target: self, action: #selector(closeButtonTapped))
+        setPrimaryActionButtonTitle()
+    }
+
+    private func setPrimaryActionButtonTitle() {
+        primaryActionButton.setTitle(createsSite ? TextContent.createSiteButton : TextContent.chooseButton, for: .normal)
+    }
+
+    private enum TextContent {
```

Comment:
Nice, good suggestion @Gio2018 and implementation @antonis. Makes things a lot cleaner!